packages/firestore/src/local/local_documents_view.ts

```diff
@@ -252,4 +264,28 @@ export class LocalDocumentsView {
         return results;
       });
   }
+
+  private getDocumentWithPendingPatches(
+    transaction: PersistenceTransaction,
+    matchingMutationBatches: MutationBatch[],
+    results: DocumentMap
```

Comment:
`results` is not very clear. Maybe something like `existingDocuments`?

Reply:
Done.